models/invite.py

```diff
@@ -43,22 +46,49 @@ def checkIsInviteInstitutionValid(data):
                 "The invite for institution have to specify the suggestion institution name")
 
     @staticmethod
+    def inviteeIsMember(invitee, institution):
```

Comment:
não tem como ser um query não? O custo desse método é alto. No for dá um load de cada membro pra procurar a igualdade.